### ISSUE TITLE
Error in parallel execution, option: - parallel: True

### ISSUE THREAD

**Issue by konstest** [saltstack/salt#46983]
### Description of Issue/Question
Hi, i gets error if i run execution states in parallel mode on WIndows minion.
I don`t check it on Linux minions & last salt-master/minions versions

### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)
```
cat parallel.sls 
dir:
  cmd.run:
    - parallel: True

sleep 5:
  cmd.run:
    - parallel: True

sleep 3:
  cmd.run:
    - parallel: True

set:
  cmd.run:
    - parallel: True
```

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)
```
salt windows-minon state.apply parallel    
windows-minon:
----------
          ID: dir
    Function: cmd.run
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1840, in call
                  ret = self.call_parallel(cdata, low)
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1716, in call_parallel
                  proc.start()
                File "c:\salt\bin\lib\multiprocessing\process.py", line 130, in start
                  self._popen = Popen(self)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 277, in __init__
                  dump(process_obj, to_child, HIGHEST_PROTOCOL)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 199, in dump
                  ForkingPickler(file, protocol).dump(obj)
                File "c:\salt\bin\lib\pickle.py", line 224, in dump
                  self.save(obj)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 67, in dispatcher
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 754, in save_global
                  (obj, module, name))
              PicklingError: Can't pickle <function create at 0x0000000008175828>: it's not found as salt.loaded.int.module.serverdensity_device.create
     Started: 11:42:34.337000
    Duration: 5641.0 ms
     Changes:   
----------
          ID: sleep 5
    Function: cmd.run
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1840, in call
                  ret = self.call_parallel(cdata, low)
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1716, in call_parallel
                  proc.start()
                File "c:\salt\bin\lib\multiprocessing\process.py", line 130, in start
                  self._popen = Popen(self)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 277, in __init__
                  dump(process_obj, to_child, HIGHEST_PROTOCOL)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 199, in dump
                  ForkingPickler(file, protocol).dump(obj)
                File "c:\salt\bin\lib\pickle.py", line 224, in dump
                  self.save(obj)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 67, in dispatcher
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 754, in save_global
                  (obj, module, name))
              PicklingError: Can't pickle <function create at 0x0000000008175828>: it's not found as salt.loaded.int.module.serverdensity_device.create
     Started: 11:42:39.978000
    Duration: 2015.0 ms
     Changes:   
----------
          ID: sleep 3
    Function: cmd.run
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1840, in call
                  ret = self.call_parallel(cdata, low)
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1716, in call_parallel
                  proc.start()
                File "c:\salt\bin\lib\multiprocessing\process.py", line 130, in start
                  self._popen = Popen(self)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 277, in __init__
                  dump(process_obj, to_child, HIGHEST_PROTOCOL)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 199, in dump
                  ForkingPickler(file, protocol).dump(obj)
                File "c:\salt\bin\lib\pickle.py", line 224, in dump
                  self.save(obj)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 67, in dispatcher
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 754, in save_global
                  (obj, module, name))
              PicklingError: Can't pickle <function create at 0x0000000008175828>: it's not found as salt.loaded.int.module.serverdensity_device.create
     Started: 11:42:41.993000
    Duration: 2250.0 ms
     Changes:   
----------
          ID: set
    Function: cmd.run
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1840, in call
                  ret = self.call_parallel(cdata, low)
                File "c:\salt\bin\lib\site-packages\salt\state.py", line 1716, in call_parallel
                  proc.start()
                File "c:\salt\bin\lib\multiprocessing\process.py", line 130, in start
                  self._popen = Popen(self)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 277, in __init__
                  dump(process_obj, to_child, HIGHEST_PROTOCOL)
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 199, in dump
                  ForkingPickler(file, protocol).dump(obj)
                File "c:\salt\bin\lib\pickle.py", line 224, in dump
                  self.save(obj)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\multiprocessing\forking.py", line 67, in dispatcher
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 425, in save_reduce
                  save(state)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 655, in save_dict
                  self._batch_setitems(obj.iteritems())
                File "c:\salt\bin\lib\pickle.py", line 687, in _batch_setitems
                  save(v)
                File "c:\salt\bin\lib\pickle.py", line 331, in save
                  self.save_reduce(obj=obj, *rv)
                File "c:\salt\bin\lib\pickle.py", line 401, in save_reduce
                  save(args)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 554, in save_tuple
                  save(element)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 606, in save_list
                  self._batch_appends(iter(obj))
                File "c:\salt\bin\lib\pickle.py", line 639, in _batch_appends
                  save(x)
                File "c:\salt\bin\lib\pickle.py", line 286, in save
                  f(self, obj) # Call unbound method with explicit self
                File "c:\salt\bin\lib\pickle.py", line 754, in save_global
                  (obj, module, name))
              PicklingError: Can't pickle <function create at 0x0000000008175828>: it's not found as salt.loaded.int.module.serverdensity_device.create
     Started: 11:42:44.243000
    Duration: 2282.0 ms
     Changes:   

Summary for windows-minon
------------
Succeeded: 0
Failed:    4
------------
Total states run:     4
Total run time:  12.188 s
```

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
salt-minion version:
```
salt windows-minon test.versions_report
windows-minon:
    Salt Version:
               Salt: 2017.7.2
     
    Dependency Versions:
               cffi: 1.10.0
           cherrypy: 10.2.1
           dateutil: 2.6.0
          docker-py: Not Installed
              gitdb: 2.0.3
          gitpython: 2.1.3
              ioflo: Not Installed
             Jinja2: 2.9.6
            libgit2: Not Installed
            libnacl: Not Installed
           M2Crypto: Not Installed
               Mako: 1.0.6
       msgpack-pure: Not Installed
     msgpack-python: 0.4.8
       mysql-python: 0.7.11.None
          pycparser: 2.17
           pycrypto: 2.6.1
       pycryptodome: Not Installed
             pygit2: Not Installed
             Python: 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)]
       python-gnupg: 0.4.0
             PyYAML: 3.11
              PyZMQ: 16.0.2
               RAET: Not Installed
              smmap: 2.0.3
            timelib: 0.2.4
            Tornado: 4.5.1
                ZMQ: 4.1.6
     
    System Versions:
               dist:   
             locale: cp1251
            machine: AMD64
            release: 2012ServerR2
             system: Windows
            version: 2012ServerR2 6.3.9600  Multiprocessor Free
```

salt-master version:
```
salt-master --versions-report
Salt Version:
           Salt: 2017.7.1
 
Dependency Versions:
           cffi: 1.11.5
       cherrypy: 11.0.0
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: 0.24.0
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: 2.18
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.24.0
         Python: 2.7.12 (default, Nov 19 2016, 06:48:10)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: Ubuntu 16.04 xenial
         locale: ANSI_X3.4-1968
        machine: x86_64
        release: 4.4.0-97-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by omgitsedwin** [comment]
Still present in 2019.2.0. Any updates on this?

**Comment by twangboy** [comment]
Looks like this is fixed on master branch (3006.2):

parallel.sls
```
dir:
  cmd.run:
    - parallel: True

powershell -c "Start-Sleep 5":
  cmd.run:
    - parallel: True

powershell -c "Start-Sleep 3":
  cmd.run:
    - parallel: True

set:
  cmd.run:
    - parallel: True
```
Returns successfully
